features/step_definitions/journey/registration_steps.rb

```diff
@@ -41,9 +55,14 @@
   if @tier == "upper"
     @journey.carrier_type_page.submit(choice: :carrier_broker_dealer)
     @carrier = "carrier_broker_dealer"
+
+    if @organisation_type == "limitedCompany"
+      @companies_house_number ||= "00445790"
```

Comment:
Check: https://github.com/DEFRA/waste-carriers-acceptance-tests/pull/269#discussion_r406257732 

Basically, if when it comes to this test the `@companies_house_number` was defined, it won't be re-defined to be the default tesco one. But if no previous step defined it, then it will be setted to be the tesco default one

Reply:
Thanks - that makes sense, thanks for explaining.